pkgs/development/compilers/llvm/common/compiler-rt/default.nix

```diff
@@ -47,13 +46,7 @@ let
   baseName = "compiler-rt";
   pname = baseName + lib.optionalString (haveLibc) "-libc";
 
-  # Sanitizers require 10.13 or newer. Instead of disabling them for most x86_64-darwin users,
-  # build them with a newer SDK and the default (10.12) deployment target.
-  apple-sdk' =
-    if lib.versionOlder (lib.getVersion apple-sdk) "10.13" then
-      apple-sdk_10_13.override { enableBootstrap = true; }
-    else
-      apple-sdk.override { enableBootstrap = true; };
+  apple-sdk' = apple-sdk.override { enableBootstrap = true; };
```

Comment:
This should be unnecessary because compiler-rt is built with the bootstrap stdenv on Darwin. This stuff was a workaround for the fact the 10.12 SDK is just too old now.

Reply:
Fixed, and confirmed that it still bootstraps.